config.go

```diff
@@ -271,7 +271,7 @@ type config struct {
 
 	Profile string `long:"profile" description:"Enable HTTP profiling on given port -- NOTE port must be between 1024 and 65535"`
 
-	UnsafeDisconnect   bool   `long:"unsafe-disconnect" description:"Allows the rpcserver to intentionally disconnect from peers with open channels. USED FOR TESTING ONLY."`
+	UnsafeDisconnect   bool   `long:"unsafe-disconnect" description:"Allows the rpcserver to intentionally disconnect from peers with open channels."`
```

Comment:
Log that it'll be deprecated in the next major release (0.10?).

Reply:
can't really log since the default value is now set to true, so we can't tell if it was set manually or not.

Reply:
Good point. We'll just need to make a note in the release notes.